frontend/src/utils/format.ts

```diff
@@ -124,6 +120,13 @@ export function formatShippingRouteName(shippingRoute: ShippingRoute) {
   return matchingRoute?.label || 'Unknown route'
 }
 
+export function formatShipmentStatus(shipmentStatus: ShipmentStatus) {
+  const matchingStatus = SHIPMENT_STATUS_OPTIONS.find(
+    (option) => option.value === shipmentStatus,
+  )
+  return matchingStatus?.label || shipmentStatus
```

Comment:
```suggestion
  return matchingStatus?.label ?? shipmentStatus
```
*nit:* use nullish coalescing operator, because it only handles cases of `null` or `undefined`, and this is the intention here.